kotsadm/web/src/components/apps/AppDetailPage.jsx

```diff
@@ -99,19 +101,29 @@ class AppDetailPage extends Component {
     this.state.getAppJob.stop();
   }
 
-  makeCurrentRelease = async (upstreamSlug, sequence) => {
-    try {
-      await fetch(`${window.env.API_ENDPOINT}/app/${upstreamSlug}/sequence/${sequence}/deploy`, {
-        headers: {
-          "Authorization": Utilities.getToken(),
-          "Content-Type": "application/json",
-        },
-        method: "POST",
+  makeCurrentRelease = (upstreamSlug, version) => {
```

Comment:
converting this from `async/await` syntax is risky, it will likely break (or probably just not wait) functions calling this with `await`.
for example: https://github.com/replicatedhq/kots/blob/b17b49e02bb07fefd745baf013ea580ff5992826/kotsadm/web/src/components/apps/AppVersionHistory.jsx#L436